Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "abci"
-version = "0.4.3"
+version = "0.4.4"
```

There are a few renamings which can be considered breaking. E.g. `BlockSizeParams` -> `BlockParams`.
Hence, a minor bump would make sense here IMHO
```suggestion
version = "0.5.0"
```